CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to UCL API
+
+First off, thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to UCL API and its packages, which are hosted in the [UCLAPI Organization](https://github.com/uclapi) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## How can I contribute?
+
+### Security vulnerability disclosure
+In case you suspect you find a vulnerability in the API, please refrain from creating an issue publicly and contact us at <isd.apiteam@ucl.ac.uk>. At the moment we do not have any bug bounty program present, but we would be happy to include you in our Hall of Fame!
+
+### Submitting bugs
+
+* **Use a clear and descriptive title** for the issue to identify the problem.
+* **Describe the exact steps which reproduce the problem** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+* **Explain which behavior you expected to see instead and why.**
+
+### Suggesting enhancements
+* **Use a clear and descriptive title** for the issue to identify the suggestion.
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
+* **Explain why this enhancement would be useful** to our API users.
+
+### Pull requests
+* Fill in [the required PR template](.uclapi/PULL_REQUEST_TEMPLATE.md).
+* Do not include issue numbers in the PR title.
```

Comment:
Yeah, but usually PRs address issues, so you can say: "Made changes X (fixes #555)"